src/core/providers/gdal/qgsgdalprovider.cpp

```diff
@@ -557,6 +557,9 @@ QString QgsGdalProvider::htmlMetadata()
       QStringList categories = QgsOgrUtils::cStringListToQStringList( GDALcategories );
       myMetadata += QgsHtmlUtils::buildBulletList( categories );
     }
+    QStringList scaleOffset;
+    scaleOffset << QStringLiteral( "Scale: %1, Offset: %2" ).arg( bandScale( i ) ).arg( bandOffset( i ) );
+    myMetadata += QgsHtmlUtils::buildBulletList( scaleOffset );
```

Comment:
shouldn't this be two separate items in the list, so they each get a bullet point?
i.e. something like:

```suggestion
    myMetadata += QgsHtmlUtils::buildBulletList( QStringList{ {
           QObject::tr("Scale: %1").arg( bandScale( i ) ),
           QObject::tr("Offset: %1").arg( bandOffset( i ) ),
    }) );
```

Reply:
I have no strong opinion here. `gdalinfo` outputs this values as a single line and I decided to apply the same approach.